src/main/java/Square.java

```diff
@@ -0,0 +1,34 @@
+public class Square extends GeometricFigure {
+    private double sideLenth;
+
+    public Square(double sideLenth, String colorOfSquare) {
+        this.sideLenth = sideLenth;
+        color = colorOfSquare;
+    }
+
+    public void draw() {
```

Comment:
```suggestion
    @Override
    public void draw() {
```
it's better to use this annotation whenever you orverride some method it's more informative